src/types/IJitsiMeetingOptions.ts

```diff
@@ -0,0 +1,83 @@
+export default interface IJitsiMeetingOptions {
+  /**
+   * The name of the room to join
+   */
+  roomName?: string;
+
+  /**
+   * The created IFrame width
+   */
+  width?: string | number;
```

Comment:
Since we are using the parentNode approach it probably doesn't make sense to expose these?

Reply:
I think it's the parentNode that we shouldn't expose, because it's overwritten when the component is loading the IFrame, as the root of the component itself acts as a 'parent node'. What do you think?

Reply:
I thought we'd always render our stuff inside a div, and use that to contain the spinner too. Is there a downside to this?

Reply:
I'm not sure I got that. The spinner is rendered inside the component and next to the only div that acts as a parent node for the IFrame. The width and height are still needed as props, because we don't want to 'hardcode' them in the component logic, is that ok?

Reply:
We should probably allow passing styles for the div we create or a class name. WDYT would make more sense here?

Reply:
or we could just expose the ref of the container div from our component, and the integrator can add any styles/classes to it. wdyt?

Reply:
How would customization work? Can you show a quick example?

Reply:
Was thinking something around the lines of:

Integrator:
```
const getMeetingDivRef  = meetingDivRef => {
   integratorMeetingRef = meetingDivRef; // integrator preserves the ref in their app.
  /// do whatver we want with the ref, including re-style
  meetingDivRef.style.backgroundColor = "#FFF';
  meetingDivRef.classList.add('integrator-class');
}

// Integrator instantiates the JitsiMeeting component with passing the callback function.
<JitsiMeeting ... getRef = {getMeetingDivRef} />
```


And inside the component once everything is loaded, we already have the `meetingRef` as the reference to the meeting's container div so we call (maybe inside the `loadIframe` function):
```
 getRef && getRef(meetingRef.current);
```

Reply:
Implemented @horymury suggestion. @saghul, thoughts?